package.json

```diff
@@ -39,5 +41,20 @@
     "name": "Amazon Web Services",
     "url": "https://aws.amazon.com"
   },
-  "version": "1.13.1"
+  "workspaces": {
+    "packages": [
+      "packages/*",
+      "packages/@aws-cdk/*",
+      "packages/@aws-cdk/*/lambda-packages/*",
+      "tools/*"
+    ],
+    "nohoist": [
```

Comment:
ooo that's tricky. What error do we get if we try to bundle a dependency in the future that will ensure this is modified?

Reply:
Well upon installing, `npm` will *warn* that a declared `bundleDependency` is, in fact, not present. That's not so great, I reckon w'll want to call `pkglint` to the rescue here?

Reply:
(Adding a `pkglint` validation for this right now).